lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -917,6 +917,53 @@ setReadOnly( Complex64Array.prototype, 'get', function get( idx ) {
 	return new Complex64( buf[ idx ], buf[ idx+1 ] );
 });
 
+/**
+* Tests whether all elements in an array pass a test implemented by a predicate function.
+*
+* @name every
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Function} predicate - test function
+* @param {*} [thisArg] - predicate function execution context
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} first argument must be a function
+* @returns {boolean} - boolean indicating whether all elements pass a test
+*
+* @example
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* function predicate( v ) {
+*     return ( realf( v ) === imagf( v ) );
+* }
+*
+* var arr = new Complex64Array( 3 );
+*
+* arr.set( [ 1.0 , 1.0 ], 0 );
+* arr.set( [ 2.0 , 2.0 ], 1 );
+* arr.set( [ 3.0 , 3.0 ], 2 );
+*
+* var bool = arr.every( predicate );
+* // returns true
+*/
+setReadOnly( Complex64Array.prototype, 'every', function every( predicate, thisArg ) {
+	var out;
+	var i;
+	if ( !isComplexArray( this ) ) {
+		throw new TypeError( 'invalid invocation. `this` is not a complex number array.' );
+	}
+	if ( !isFunction( predicate ) ) {
+		throw new TypeError( format( 'invalid argument. First argument must be a function. Value: `%s`.', predicate ) );
+	}
+	for ( i = 0; i < this._length; i++ ) {
+		out = predicate.call( thisArg, this.get( i ), i, this );
```

Comment:
```suggestion
		out = predicate.call( thisArg, this.get( i ), i, this );
```

This is fine, for now. Making a note that we should implement a private `_get` method to avoid unnecessarily performing validation checks for each element in this loop.